packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -90,14 +88,14 @@ const ProfilePanel: FC = () => {
     }, [
         sendVerificationEmailError,
         data,
-        error,
+        updateUserError,
         showError,
         isEmailServerConfigured,
     ]);
 
-    const handleUpdate = () => {
+    const onSubmit = form.onSubmit(({ firstName, lastName, email }) => {
```

Comment:
I personally prefer to have `handle` prefix on local props and `on` prefix to passed components. wdyt?

Reply:
when reading code: if you are seeing `onSave={onSave}` it instantly means that you are using a prop from above. `onSave={handleSave}` means you are using something that's declared locally. our codebase is not consistent ATM but making progress one by one

Reply:
🤦 agree completely, why didn't I follow that. Let me rename these!